src/common/profile.c

```diff
@@ -0,0 +1,351 @@
+/**
+ * (C) Copyright 2020 Intel Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+ * The Government's rights to use, modify, reproduce, release, perform, display,
+ * or disclose this software are subject to the terms of the Apache License as
+ * provided in Contract No. B609815.
+ * Any reproduction of computer software, computer software documentation, or
+ * portions thereof marked with this legend must also reproduce the markings.
+ */
+/**
+ * This file is part of the DAOS profile.
+ */
+#define D_LOGFAC	DD_FAC(common)
+
+#include <abt.h>
+#include <daos/common.h>
+#include <daos_errno.h>
+#include <gurt/list.h>
+
+#define DEFAULT_CHUNK_SIZE  10240
+#define DEFAULT_CHUNK_CNT_LIMIT	100
+
+char *profile_op_names[] = {
+	[OBJ_PF_UPDATE_PREP] = "update_prep",
+	[OBJ_PF_UPDATE_DISPATCH] = "update_dispatch",
+	[OBJ_PF_UPDATE_LOCAL] = "update_local",
+	[OBJ_PF_UPDATE_END] = "update_end",
+	[OBJ_PF_UPDATE_WAIT] = "update_end",
```

Comment:
It seems like OBJ_PF_UPDATE_WAIT should be "update_wait" instead in order to be consistent with the other profile names.   After reading more, I see that this is an old definition that moved here.  Still, the definition seems like it has an error.  Can you double check to see if this is using "update_end" is actually intended?

Reply:
Ah, yes. I will fix it. Thanks.